api/curl/upload-snapshot-cron.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Start hit the reporting snapshot & upload cron endpoint  locally.
```

Comment:
I'm not clear on what is intended by this sentence.  Also, could you rename this script to include "local" somewhere?